### DISCUSSION TITLE
Template quarto

### DISCUSSION THREAD

**Discussion by MaelTheuliere** [spyrales/gouvdown#54] [Ideas]
Hello, quarto est en train de supplanter Rmarkdown en terme de popularité, et il y a pas mal de bonnes raisons à cela. Est ce que cela vaudrait pas le coup de faire une version quarto des templates {gouvdown} ?

**Comment by tvroylandt** [comment]
Concernant pagedown, compatibilité annoncée pour cet été donc pour le moment non

A court terme je pense qu'on a le temps de voir venir

**Reply by MaelTheuliere** [reply]
pour pagedown je suis pas sur de comprendre...
Je parlais de développer des extensions quarto pour les sorties existantes (website, book, html)

**Reply by tvroylandt** [reply]
Ils vont faire un paged quarto dans l'été, c'est tout.

Pour les sorties existantes, pareil, on a un sacré temps avant que Rmd soit dépassé. Et ne serait-il pas intéressant de dev les extensions quarto dans un package non R ?

**Reply by MaelTheuliere** [reply]
Ok je comprends mieux. En effet,  la gestion des extensions quarto ne se fait pas dans un package R. Deux exemples  :
- https://github.com/InseeFrLab/onyxia-quarto
- https://github.com/hrbrmstr/quarto-organization-template 

Et oui, Rmarkdown n'est pas mort !
